src/Strategy/BerkeleyDB.hs

```diff
@@ -72,9 +72,21 @@ findProjects osInfo = walkWithFilters' $ \dir _ files -> do
   case findFileNamed "Packages" files of
     Nothing -> pure ([], WalkContinue)
     Just file -> do
-      if (Text.isInfixOf "var/lib/rpm/" $ toText . toFilePath $ file)
+      if isSupportedPath file
         then pure ([BerkeleyDatabase dir file osInfo], WalkContinue)
         else pure ([], WalkContinue)
+  where
+    -- The standard location for this is '/var/lib/rpm/',
```

Comment:
It looks like the new directory is something the Fedora and distros downstream of it are doing: https://fedoraproject.org/wiki/Changes/RelocateRPMToUsr

Would it be worthwhile to just check for just the one alternate case? It looks like the change isn't arbitrary and chosen by distros either, it has buy-in from upstream `rpm`: http://lists.rpm.org/pipermail/rpm-ecosystem/2021-December/000781.html

I only looked into this/ask because searching for a path with `rpm` in it feels a little bit arbitrary. Assuming I had a filename of `Packages` or `rpmdb.sqlite` and got a `False` out of `isSupportedPath` I don't think I would really be that confident that it was _not_ an rpm db.

Reply:
Since we already do it this way today, are you okay with leaving this as-is? Context: I want to reduce the uncertainty of this change as much as possible, and I think the best way to do that is to just exactly copy what we're already doing for the NDB strategy.

Reply:
I am, thanks!